### PR TITLE
Deleted

### PR DESCRIPTION
**Pull request by Rexadev**

### CODE REVIEW

File: porn [hunk 1]
```diff
@@ -367,3 +367,10 @@
 0.0.0.0 yamidas.yamiboard.com
 0.0.0.0 yamigama.com
 0.0.0.0 yespornplease.com
+0.0.0.0 gfycat.com
+#https://gfycat.com/gifs/search/boobs
+0.0.0.0 desixossip.com
+0.0.0.0 giphy.com
+0.0.0.0 giphy-com.cdn.ampproject.org
+0.0.0.0 media0-giphy-com.cdn.ampproject.org
+#https://giphy.com/search/boobs
```
Comment (spirillen) [inline porn:376]: I don't see any nudity :eyeglasses:   like at all

Comment (Rexadev) [reply porn:376]: https://giphy.com/gifs/latchedlouisville-breastfeeding-nursing-latch-hatch-8WqeRjbsdHwqexvVIr
https://giphy.com/gifs/breastfeeding-Rib9PWf1KcnMn8XNtQ
https://giphy.com/gifs/stunning-FMJIaIm1CWbss [looks like cheese]

Comment (spirillen) [reply porn:376]: A baby licking on a elbow is porn? `https://giphy.com/gifs/latchedlouisville-breastfeeding-nursing-latch-hatch-8WqeRjbsdHwqexvVIr`

So now it is porn when you works down the street a summer day?? `https://giphy.com/gifs/stunning-FMJIaIm1CWbss`

Just not nudity/porn or anything alike, it comedy: `https://giphy.com/gifs/breastfeeding-Rib9PWf1KcnMn8XNtQ`

Comment (Rexadev) [reply porn:376]: You would not want a 7 year old see it so it is porn. For me bikini/deep cleavage/side boob/ass crack/mens low pant "V" is porn

Comment (spirillen) [reply porn:376]: well my kinds have been used to see people in bikini's on the beach and streets and no kids I know would be harmed in any way for any of the examples.

If @brianclemens choses to merge that, fine with me, but there are not going to mypdns blacklists.

**Pull request closed without merging**